Eatery/Views/BRB/FavoriteTableViewCell.swift

```diff
@@ -0,0 +1,61 @@
+//
+//  FavoriteTableViewCell.swift
+//  Eatery
+//
+//  Created by Noah Pikielny on 4/12/21.
+//  Copyright © 2021 Cornell AppDev. All rights reserved.
+//
+
+import UIKit
+
+class FavoriteTableViewCell: UITableViewCell {
+
+    private let nameLabel = UILabel()
+    private let servingLabel = UILabel()
```

Comment:
From what I remember on Eatery's codebase, UIKit components only have their types declared in the class field section.  For example

```
private let nameLabel: UILabel!
```
instead of
```
private let nameLabel = UILabel()
```
and then you set the nameLabel within init.  If this isn't a universal thing, it's not a big deal.  But, if it is, I think you should change this to stay consistent with the formatting of the rest of the codebase.

Reply:
I disagree with this. I much prefer avoiding the unwrapped optionals (!). It's your choice though.